### ISSUE TITLE
Considerations on T.PA_Enrolment

### ISSUE THREAD

**Issue by BrJu** [biometricITC/cPP-biometrics#27]
From @yamadaAIST

The current [T.PA_Enrolment](https://github.com/nils-tekampe/cPP-biometrics/blob/c1c449a117e0d7c4483b4aa0c2d266da9be65ee7/input/biometricCPP.xml#L226) is as follows:

An attacker may attempt to get impersonated as another user during enrolment. In order to perform the attack, the attacker uses artificial biometric characteristics, carrying the biometric characteristic of the attacked user (as so called Presentation Attack) 

The current T.PA_Enrolment contains two threats: one is identity theft and the other is presentation attack. 
The former half is identity theft. If an attacker uses another user’s identity and enrols the attacker's own biometric characteristic, then he can impersonate the user of the identity later. 
The latter half is presentation attack. Even if an attacker doesn’t know another user’s identity, he or she can enrol an artefact with his or her identity and later ask another attacker to impersonate the first attacker using the artefact. This makes a collusion attack. For example, the first attacker can claim that his smartphone was stolen and his account was illegally used to pay for an Internet shopping through a presentation attack.

An impersonation is an attack to authentication system, i.e., it cannot be done if a user is not registered. Still presentation attack is possible to enrolment system, for example, an attempt to enrol artefacts or enrol a biometric characteristic in such a way that it results in a biometric reference in a low quality. They shall not make biometric references. They are considered as presentation attacks to enrolment system. If they are successfully enrolled, they will facilitate presentation attacks to verification system later.
In biometrics, identity is out of scope. By definition, a biometric verification is just a comparison of a biometric probe against a biometric reference and does not handle the identity.  In BioAPI, a standard API in biometrics specified in ISO/IEC JTC 1/SC 37, there is no input parameter of identity for the verification function. Threats in the cPP should be as universal as possible, Therefore the threat on identity theft should be optional.
In conclusion, the current threat should be divided into a threat of presentation attack and an optional threat of identity theft.

T.PA_Enrolment 
During enrolment, an attacker may attempt to present an artefact or his/her biometric characteristic in such a way that it results in a biometric data of a low quality to be enrolled as his/her biometric reference for the purpose of successful presentation attacks of later biometric verification.

T.IdTheft_Enrolment (optional)
During enrolment, an attacker may attempt to use another user’s identity for the purpose of impersonation of later biometric authentication.
